server/server.js

```diff
@@ -170,6 +183,36 @@ app.post('/api/tripUpdate', (req, res) => {
     });
 });
 
+app.post('/api/resetPassword', (req, res) => {
+    const newPassword = uuid();
```

Comment:
In the future maybe would be better to generate a more user-friendly password, but at the moment the most important is the functionality working :D